akka-service-locator/core/src/main/resources/reference.conf

```diff
@@ -33,7 +33,41 @@ lagom.akka.discovery {
   service-name-mappings {
   }
 
+  # Static mappings (config based mappings). Each mapping should define the following:
+  # - endpoints - A list of pairs (host, port) where:
+  #   * host - hostname
+  #   * port - network port number
+  #
+  # For example:
+  # static-mappings {
+  #  my-service-name {
+  #    endpoints = [
+  #      {
+  #        host: myhost1
+  #        port: 9101
+  #      },
+  #      {
+  #        host: myhost2
+  #        port: 9101
+  #      },
+  #      ...
+  #    ]
+  #  }
+  # }
+  static-mappings {
+  }
 
   # The timeout for a successful lookup.
   lookup-timeout = 5 seconds
 }
+
+akka {
+  discovery {
+    method = aggregate
+    aggregate {
+      discovery-methods = ["config", "akka-dns"]
+    }
+
+    config.services = ${lagom.akka.discovery.static-mappings}
```

Comment:
I'd rather not introduce the `static-mappings` indirection and, instead, I'd prefer just using the vanilla feature. How about just adding:

```
akka {
  discovery {
    method = aggregate
    aggregate {
      discovery-methods = ["akka-dns", "config"]
    }
  }
}
```

and then documenting (in `docs/`) how to add config-based blocks linking to https://doc.akka.io/docs/akka/current/discovery/index.html ?

Reply:
@ignasi35 `["akka-dns", "config"]` or `["config", "akka-dns"]`?
@renatocaval in https://github.com/lagom/lagom-akka-discovery-service-locator/issues/81#issuecomment-486631635 proposed `["config", "akka-dns"]`.

Reply:
Good catch! I think `["config", "akka-dns"]` is correct. I just copy/pasted the sample from the Akka docs were the order is the opposite.

Reply:
done :)